src/App.tsx

```diff
@@ -1,26 +1,21 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import {BrowserRouter as Router,Routes,Route} from  'react-router-dom'; 
+import SingUp from './pages/SignUp';
+import LogIn from './pages/LogIn';
 
-function App() {
+
+
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="min-h-screen flex flex-col justify-center">
+      <Router>
+        <Routes>
+          <Route path='/' element={<SingUp/>}/>
```

Comment:
So if someone is logged in and they try to hit the "/", what happens?

Reply:
The sign up page comes up.

Reply:
I am not sure I get the question